src/DIRAC/Core/Tornado/Server/BaseRequestHandler.py

```diff
@@ -0,0 +1,783 @@
+""" BaseRequestHandler is the base class for tornados services and etc handlers.
```

Comment:
I did not read the full file, but what was wrong with the existing `TornadoService` ?

Reply:
it's great, but for RESTful endpoints it was a bit unsuitable, so I decided to move the basic logic to `BaseRequestHandler` to leave the `TornadoService` logic unchanged, but create a `TornadoREST` based on the same logic to describe handlers with a different API.
There may be a better way, but I thought the solution was the easiest

Reply:
I added more description in BaseRequestHandler, TornadoREST

Reply:
`BaseRequestHandler`, `TornadoREST`, `TornadoService` was updated